docs/BreakingChanges/131 PreviewLostKeyboardFocus is called repeatedly if its handler shows a message box.md

```diff
@@ -24,10 +24,10 @@ There are two options to work around this issue -
 2. It may be avoided by showing the message box from a `LostKeyboardFocus` event handler (as opposed to a `PreviewLostKeyboardFocus` event handler).
 
 ### Affected APIs
-* `M:System.Windows.ContentElement.add_PreviewLostKeyboardFocus(System.Windows.Input.KeyboardFocusChangedEventHandler)`
-* `M:System.Windows.IInputElement.add_PreviewLostKeyboardFocus(System.Windows.Input.KeyboardFocusChangedEventHandler)`
-* `M:System.Windows.UIElement.add_PreviewLostKeyboardFocus(System.Windows.Input.KeyboardFocusChangedEventHandler)`
-* `M:System.Windows.UIElement3D.add_PreviewLostKeyboardFocus(System.Windows.Input.KeyboardFocusChangedEventHandler)`
+* `E:System.Windows.ContentElement.PreviewLostKeyboardFocus`
+* `E:System.Windows.IInputElement.PreviewLostKeyboardFocus`
+* `E:System.Windows.UIElement.PreviewLostKeyboardFocus`
+* `E:System.Windows.UIElement3D.PreviewLostKeyboardFocus`
 
```

Comment:
Flagging the event will alert people who invoke the event in addition to just those who attach handlers. This is a tricky issue to detect, so neither of those is perfect but I think only flagging cases when a handler is attached might be better scoped.